Jenkinsfile

```diff
@@ -4,25 +4,36 @@
 properties([buildDiscarder(logRotator(artifactNumToKeepStr: '20', numToKeepStr: '20'))])
 
 node {
-
+  stage 'Setup'
```

Comment:
Using the 'stage' step without a block argument is deprecated. See [blog post here](https://jenkins.io/blog/2016/10/16/stage-lock-milestone/). Any reason why its still used here?